elide-model-config/src/main/java/com/yahoo/elide/modelconfig/model/Table.java

```diff
@@ -51,7 +50,6 @@
 @Data
```

Comment:
Use builder for Jackson deserializer
```suggestion
@JsonDeserialize(builder = Table.TableBuilder.class)
@Data
```